packages/rollup-tests/package.json

```diff
@@ -14,7 +14,8 @@
     "fs-extra": "^11.2.0",
     "mocha": "^10.4.0",
     "rolldown": "workspace:*",
-    "source-map-support": "^0.5.21"
+    "source-map-support": "^0.5.21",
+    "oxc-transform": "0.35.0"
```

Comment:
What's the usage of this?

Reply:
Used to prettier code to diff snapshot code